packages/neuron-wallet/src/block-sync-renderer/sync/light-connector.ts

```diff
@@ -75,30 +75,36 @@ export default class LightConnector extends Connector<CKBComponents.Hash> {
     const fetchTxHashes = fetchCellDeps
       .map(v => v.outPoint.txHash)
       .map<[string, string]>(v => ['fetchTransaction', v])
-    let txs = await this.lightRpc
-      .createBatchRequest<any, string[], (CKBComponents.TransactionWithStatus | null)[]>(fetchTxHashes)
+    const txs = await this.lightRpc
+      .createBatchRequest<any, string[], (CKBComponents.TransactionWithStatus | { status: 'fetching' | 'added' | 'not_found' })[]>(fetchTxHashes)
       .exec()
-    if (txs.some(v => !v)) {
-      // if some txs fetch to added, then fetch the actural txs
-      txs = await this.lightRpc
-        .createBatchRequest<any, string[], (CKBComponents.TransactionWithStatus | null)[]>(fetchTxHashes)
-        .exec()
+    if (txs.some(v => 'status' in v && !!v.status)) {
```

Comment:
In which case, the `v.status` is `undefined` while the field exists

Reply:
There is no case for that. I change the return type of `fetchTransaction` by https://github.com/nervosnetwork/neuron/pull/2675/commits/25a753daf8e9a5a76e2eeb0aebaacc628c72eab2 and I think I should keep all the status of `fetch_transaction`'s results. Then it will be more clear.